cocos/asset/assets/image-asset.ts

```diff
@@ -523,43 +485,52 @@ export class ImageAsset extends Asset {
     @override
     get _nativeAsset (): any {
         // Maybe returned to pool in webgl.
-        return this._nativeData;
+        return this._imageData.data;
     }
     // TODO: Property 'format' does not exist on type 'ImageBitmap'
     // set _nativeAsset (value: ImageSource) {
     set _nativeAsset (value: any) {
-        if (!(value instanceof HTMLElement) && !isImageBitmap(value)) {
-            value.format = value.format || this._format;
-        }
         this.reset(value);
     }
 
+    /*
+     * @en Raw image data.
+     * @zh 原始图像数据。
+     */
+    get rawData (): any {
+        return this._imageData.rawData();
+    }
+
     /**
-     * @en Image data.
-     * @zh 此图像资源的图像数据。
+     * @en Image data source(include: HTMLCanvasElement | HTMLImageElement | IMemoryImageSource | ImageBitmap | ArrayBufferView).
+     * @zh 图像的来源（包括：HTMLCanvasElement | HTMLImageElement | IMemoryImageSource | ImageBitmap | ArrayBufferView）。
      */
-    get data (): ArrayBufferView | HTMLCanvasElement | HTMLImageElement | ImageBitmap | null {
-        if (isNativeImage(this._nativeData)) {
-            return this._nativeData;
-        }
+    get data (): any {
```

Comment:
Why return any? We should avoid using any.

Reply:
As it returns `ImageData`, and `data` is contained in `ImageData`. So is this API needed?

Reply:
It is needed here because in the web platform, imagesource(HTMLCanvasElement | HTMLImageElement) is used directly. 

In native platform is used raw image data.